pydantic_ai_slim/pydantic_ai/_utils.py

```
@@ -540,9 +540,25 @@ def get_traceparent(x: AgentRun | AgentRunResult | GraphRun[Any, Any, Any]) -> s
+        if not f.repr:
+            continue
+        val = getattr(self, f.name)
+        if f.default is not MISSING:
+            try:
+                show = bool(val != f.default)
+            except Exception:
+                show = True
+        elif f.default_factory is not MISSING:
+            try:
+                show = bool(val != f.default_factory())
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)